cardano-db-sync/src/Cardano/DbSync/Api/Types.hs

```diff
@@ -83,10 +83,13 @@ data FixesRan = NoneFixRan | DataFixRan | AllFixRan
 data ConsistentLevel = Consistent | DBAheadOfLedger | Unchecked
   deriving (Show, Eq)
 
-data ExtraMigrations = ExtraMigrations
-  { emRan :: Bool
-  , emConsume :: Bool
-  , emPrune :: Bool
+data PruneConsumeMigration = PruneConsumeMigration
+  { pcmRan :: Bool
+  , pcmConsumeFlag :: Bool
```

Comment:
This flag is never used I think it can be removed now that `pcmConsumeOrPruneTxOut` is introduced. Also since this type is about a migration and not about user flags, I slightly prefer the previous name. `Consume` shows the existance of the `tx_out.consumed_by_tx_in_id` field which exists when the user has set any of these two flags.

Reply:
cool I'll take the flag part out 👍